### DISCUSSION TITLE
Active Directory - Tenant Selector?

### DISCUSSION THREAD

**Discussion by rndthoughts** [requarks/wiki#6211] [Help / Questions]
Hi all,

Got Wikijs working fine with Azure AD for those users who only belong to one tenant.  On my machine, I have several different tenants signed in at once and usually when I auth with AD, i get a tenant selector:

![image](https://user-images.githubusercontent.com/4190823/221520851-9695da88-e7e3-4fff-9557-9981495bcfc4.png)

However when i attempt to login with AD in WikiJs, it seems to select the last tenant I logged in with and doesnt give me the above options.  Is there a way to configure this as it prevents me logging in unless i launch an "InPrivate" window and manually login every time with the right domain.

Steve

**Comment by lhuaraya** [comment]
Hi, I'm luigi and I work as DevOps at Laude.
I have also activated the login with Azure AD but I have the problem of **Invalid email or Password**.
![image](https://user-images.githubusercontent.com/112864435/222713736-42d0517f-a80b-4efc-91a9-e44844644623.png)
How have you avoided this problem? Changing something in the code of the Web Application?

**Reply by rndthoughts** [reply]
The only reason it happened to me is if Windows is signed into multiple Active Directory Tenants (mainly, those that appear in your "Access Work or School list in Settings -> Account).  If you open an "InPrivate" browser window and login it works.  At the moment, I use multiple Microsoft Edge profiles, each one signed in with a different tenant and the "Use My Current Account" extension to force the browser to use the primary tenant in each profile.
